### ISSUE TITLE
Build Report Tab showing HTTP 404

### ISSUE THREAD

**Issue by PaulAdams-Conti** [gitextensions/gitextensions#6067]
## Current behaviour

Build report tab is not showing Jenkins report. Works fine when clicking on "Open build report in the browser" in the context menu of the commit. Maybe something to do with password protected job, but user and password were entered when requested.

## Expected behaviour

Jenkins build report should be shown in Build report tab

## Steps to reproduce

1. Build server integration is set to Jenkins
1. Jenkins Server URL is entered
1. Project name is entered
1. User name and password were asked for and input (but I can't find any way to check that or change it now, as it is no longer asked for and I don't know where it is stored)
1. Build was triggered by webhook and green dot appears next to commit
1. Opening Build Report tab results in HTTP 404 message

## Screenshots

![image](https://user-images.githubusercontent.com/20796832/50839447-9f2faa00-1360-11e9-9c1f-cf2e7620535e.png)

![image](https://user-images.githubusercontent.com/20796832/50888792-bb7e2600-13f6-11e9-9d57-aa4add29bb04.png)

## Did this work in previous version of GitExtensions

Don't know, was just trying out the feature for the first time

## Environment

- Git Extensions 3.00.00.4433
- Build fca7cf228b481ee8c1b779cf7b882ccdfbdcd1bc
- Git 2.19.1.windows.1
- Microsoft Windows NT 10.0.16299.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

**Comment by drewnoakes** [comment]
@PaulAdams-Conti do you mind me asking whether you saw a template when creating this issue? We recently merged #5955 and I'm concerned people aren't seeing issue templates.

If you did see it and ignored it, please complete it.

**Comment by gerhardol** [comment]
You definitely need to provide what build server and the actual configuration too.

**Comment by PaulAdams-Conti** [comment]
@drewnoakes no there was no template when I created the issue

**Comment by PaulAdams-Conti** [comment]
Just checked and it is definitely to do with enabling Project Based Security in the Jenkins Job. When that is disabled, build report is shown correctly.

**Comment by PaulAdams-Conti** [comment]
@RussKie I'm not sure how the password that I entered is being used in the code to access the Jenkins job URL. The fact that the user and password are asked for the first time I set up the build server integration suggests that it should be being used to access the Jenkins job, but for some reason isn't working.

Maybe there could be a Test Connection button and user and password fields that can optionally be entered on the "Build server integration" page and maybe some diagnostic information in the log to show the URL that has been generated (including any name and password).

I still don't know where the user and password that was asked for is being saved and if there aren't any fields for this, how is it possible to change them once they have been entered the first time?
